Locker/Locker/Locker.swift

```diff
@@ -168,21 +192,32 @@ public extension Locker {
 
     @objc(resetForUniqueIdentifier:)
     static func reset(for uniqueIdentifier: String) {
-        Locker.userDefaults?.removeObject(forKey: LockerHelpers.keyDidAskToUseBiometricsIDForUniqueIdentifier(uniqueIdentifier))
-        Locker.userDefaults?.removeObject(forKey: LockerHelpers.keyShouldAddSecretToKeychainOnNextLoginForUniqueIdentifier(uniqueIdentifier))
-        Locker.userDefaults?.removeObject(forKey: LockerHelpers.keyBiometricsIDActivatedForUniqueIdentifier(uniqueIdentifier))
+        Locker.userDefaults?.removeObject(
+            forKey: LockerHelpers.keyDidAskToUseBiometricsIDForUniqueIdentifier(uniqueIdentifier)
+        )
+        Locker.userDefaults?.removeObject(
+            forKey: LockerHelpers.keyShouldAddSecretToKeychainOnNextLoginForUniqueIdentifier(uniqueIdentifier)
+        )
+        Locker.userDefaults?.removeObject(
+            forKey: LockerHelpers.keyBiometricsIDActivatedForUniqueIdentifier(uniqueIdentifier)
+        )
         Locker.deleteSecret(for: uniqueIdentifier)
     }
 }
 
 private extension Locker {
     // added a function with a closure since we can't throw errors
     // from async threads which is throwable
-    private static func setSecretForDevice(_ secret: String, for uniqueIdentifier: String, _ completion: @escaping ((LockerError?) throws -> Void)) {
-        let query: [CFString : Any] = [
-            kSecClass : kSecClassGenericPassword,
-            kSecAttrService : LockerHelpers.keyKeychainServiceName,
-            kSecAttrAccount : LockerHelpers.keyKeychainAccountNameForUniqueIdentifier(uniqueIdentifier)
+    // swiftlint:disable:next function_body_length
```

Comment:
Maybe consider breaking this method into a few (private) small ones? :)

Reply:
It didn't give out a warning anymore since we removed do-catch blocks but I still extracted one function from it 😄